src/AndcultureCode.CSharp.Sitefinity.Core/Extensions/DynamicContentExtensions.cs

```diff
@@ -86,6 +86,12 @@ public static T MapTo<T>(this DynamicContent content, string includes = null) wh
                 // so we have to handle them differently than other cases.
                 if (property.PropertyType.IsEnum && dynamicProperty.PropertyType == typeof(string))
                 {
+                    var dynamicPropertyValue = dynamicProperty.GetValue(content);
+                    if (dynamicPropertyValue == null)
+                    {
+                        continue;
+                    }
+
                     var value = Enum.Parse(property.PropertyType, dynamicProperty.GetValue(content).ToString());
```

Comment:
Technically you could set the second param to `dynamicPropertyValue.ToString()` since you evaluated it on line 89.

Reply:
Crap... that's what I meant to do.  I'll update.  Good catch.